### ISSUE TITLE
Night Elf Passive Affects Spells

### ISSUE THREAD

**Issue by Skipslegdays** [dalaranwow/dalaran-wow#5977]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Night elf passive 2% miss chance applies to spells and melee/ranged attacks

**Expected behaviour**: Tell us what should happen instead

Night elf passive 2% miss chance applies to melee/ranged attacks only

**Steps to reproduce the problem**:

1. Have a character with 4% spell hit
2. Cast spells at a night elf
3. Eventually one will miss

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

No proof needed that it happens (test it yourself) or that it's not supposed to happen (every knows it shouldn't apply to spells). Honestly idk why I ever bother logging the bug since nothing gets fixed here

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
